recipes/master.rb

```diff
@@ -20,6 +20,15 @@
   action :enable
 end
 
+user node['salt']['master']['api']['user']['name'] do
```

Comment:
Agreed to put `user` resource in a block `if default['salt']['master']['api']['user']['enable'] == true` in favor of Chef13.

For scenarios where user already exists, it is still possible to change the user name by setting `default['salt']['master']['api']['user']['name']`. But, it would not matter if wrapped around an `if` block.